installer/deb/create_deb.sh

```diff
@@ -89,7 +89,8 @@ Maintainer: Massa Labs <massa.net>
 Homepage: https://station.massa.net
 Description: An entrance to the Massa blockchain.
     MassaStation is a secured gateway to the Massa blockchain. This application provides a user-friendly way to access, use and build on the Massa blockchain while keeping you safe from the dangers of the internet.
-Depends: curl, iproute2, libnss3-tools, debconf (>= 0.5) | debconf-2.0
+Depends: debconf (>= 0.5) | debconf-2.0
```

Comment:
Are we sure we can remove all those dep ? 
For example, `curl` might not be useful anymore, but don't we need `libnss3-tools` and `iproute2` to generate https certificates ? 🤔